[23.0] 23.0 Release notes (#15354)
galaxyproject/galaxy: doc/source/releases/23.0_announce_user.rst

* Misc fixes 202209
  (thanks to `@nsoranzo <https://github.com/nsoranzo>`__).
  `Pull Request 14592`_
* Type annotations for base toolbox functionality.
  (thanks to `@jmchilton <https://github.com/jmchilton>`__).
  `Pull Request 14629`_
* Simplify head wrapper - same functionality, no pipe
  (thanks to `@wm75 <https://github.com/wm75>`__).
  `Pull Request 14895`_
* Update interactivetool_ml_jupyter_notebook.xml
  (thanks to `@anuprulez <https://github.com/anuprulez>`__).
  `Pull Request 14910`_
* Add interactive tool for Mgnify Jupyter lab
  (thanks to `@bebatut <https://github.com/bebatut>`__).
  `Pull Request 14950`_
*  Add interactive tool for Pavian
  (thanks to `@bebatut <https://github.com/bebatut>`__).
  `Pull Request 15068`_
* Fix add value tool, bump linter level for ``format="input"`` to error
  (thanks to `@mvdbeek <https://github.com/mvdbeek>`__).
  `Pull Request 15304`_
* Add linting with ruff
  (thanks to `@nsoranzo <https://github.com/nsoranzo>`__).
  `Pull Request 15315`_

Release Testing Team
===========================================================

A special thanks to the release testing team for testing many of the new features and reporting many bugs:

> <team members go here>

Reviewer: ```suggestion
Alireza Heidari <https://github.com/itisAliRH>
Bryan Raubenolt <https://github.com/thepineapplepirate>
Cameron Hyde <https://github.com/neoformit>
Jennifer Hillman-Jackson <https://github.com/jennaj>
John Chilton <https://github.com/jmchilton>
John Davis <https://github.com/jdavcs>
Laila Los <https://github.com/ElectronicBlueberry>
```

Drag and Drop Between Histories
-------------------------------

With the new advent of the new Multi-History mentioned above, we now also can drag and drop between histories! And no longer are you restricted from dragging into the current history, instead you can drag files back and forth between all histories as well.

.. raw:: html

   <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/8xzSzd2TOMY" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen alt="Video of a user using the new search interface to find collection operations tools, and then a Filter tool."></iframe>

Change the Datatype of a Collection
-----------------------------------

It's easier than ever to change the datatype of every dataset in a collection using the pencil icon. Previously this could be achieved through the Apply Rules tool, however now it can be done just like with individual datasets.

.. raw:: html

>    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/SNvne6GdKzs" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen alt="Video of a user using the new search interface to find collection operations tools, and then a Filter tool."></iframe>

Reviewer 2: ```suggestion
   <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/SNvne6GdKzs" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen alt="Video of a user changing the datatype for a dataset collection."></iframe>
```

Reviewer 3: yeah good improvement.

   <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/SNvne6GdKzs" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen alt="Video of a user using the new search interface to find collection operations tools, and then a Filter tool."></iframe>

RO-Crate / Biocompute Workflow Invocation Export
----------------------------

> Galaxy now offers two options for packaging and publishing workflow artifacts: RO-Crate and BioCompute Object. These standard-based formats provide a way to bundle research data and metadata in a structured way. To learn more about these formats, check out RO-Crate <https://www.researchobject.org/ro-crate/>__ and BioCompute Objects <https://biocomputeobject.org/>__. Please note that you'll need to enable the Celery task system in order to export to both formats. Additionally, you can directly export these files to any writeable remote file source that has been configured (e.g., FTP or Dropbox).

Reviewer 3: I'm not sure about the added sentence, the audience isn't admins, they don't care about celery.

```suggestion
Galaxy now offers two options for packaging and publishing workflow artifacts: RO-Crate and BioCompute Object. These standard-based formats provide a way to bundle research data and metadata in a structured way. To learn more about these formats, check out `RO-Crate <https://www.researchobject.org/ro-crate/>`__ and `BioCompute Objects <https://biocomputeobject.org/>`__. You can directly export these files to any writeable remote file sources that have been configured (e.g., FTP or Dropbox).
```

the ` are additionally important for the urls, no?

History Export Tracking
----------------------------

> Good news for users with Celery enabled on their Galaxy instances: a new History export user interface is now available with additional options for generating temporal History downloads. This feature also allows you to package your history as an RO-Crate and export it to remote file sources for more permanent storage. Plus, every time you export your history to one of these remote sources, a new tracking record is created with a button that lets you re-import that history snapshot with ease.

Reviewer 3: Could you add a screenshot here?

```suggestion
A new History export user interface is now available that makes it easier to download your histories. You can export your history in a variety of formats, and either download it directly, or save it to a remote file source for more permanent storage. Plus, every time you export your history to one of these remote sources, a new tracking record is created with a button that lets you re-import that history snapshot with ease.
```

again I think we can just say this without confusing users over what celery is or how they could enable it. I think we should pair this with a note in the admin facing release notes that mentions Celery is required for these new features that your users might be asking about.

Reviewer 4: You're right, I was just a bit anxious about users being confused because these features are not visible in big instances like main.

Reviewer 2: We'll have celery on main pretty soon, I think it's not worth stressing about this.
